generators/client/templates/common/README.md.jhi.client.ejs

```diff
@@ -149,6 +149,36 @@ Note: There are still a few other things remaining to do for Leaflet that we won
 
 For further instructions on how to develop with JHipster, have a look at [Using JHipster in development][].
 
+<%_ if (microfrontend) { -%>
+### Developing Microfrontend
+
+Microservices doesn't contain every required backend feature to allow microfrontends to run alone.
+You must start a pre-built gateway version or from source.
+
+Start gateway from source:
+
+```
+cd gateway
+npm run app:start
```

Comment:
Since we tend to target Java developers, I think it'd be good to add a comment here that says `./gradlew` or `./mvnw` is the same command. Especially since they tend to be shorter. I'm just not sure how to word it. Maybe something like:

```
npm run app:start # you can also use ./mvnw or ./gradlew
```

Reply:
In the documentation we always add both commands.
I always wonder if we should start putting generic npm commands and the user can easily do `cat package.json | grep app:start`.
Or even generate the scripts at the documentation since our npm scripts should be stable at each jhipster major version.

Anyway this discussion is not related to this PR.

Reply:
Updated.